src/ILVerify/src/Program.cs

```diff
@@ -168,31 +168,30 @@ private void PrintResult(VerificationResult result, EcmaModule module, string pa
             PrintMethod(method);
             Write("]");
 
-            var args = result.Error;
-            if (args.Code != VerifierError.None)
+            if (result.Code != VerifierError.None)
             {
                 Write("[offset 0x");
-                Write(args.Offset.ToString("X8"));
+                Write(result.GetArgumentValue<int>("Offset").ToString("X8"));
                 Write("]");
 
-                if (args.Found != null)
+                if (result.GetArgumentValue<string>("Found") != null)
                 {
                     Write("[found ");
-                    Write(args.Found);
+                    Write(result.GetArgumentValue<string>("Found"));
```

Comment:
It would be nice to avoid the redundant GetArgumentValue calls.

Reply:
added `Try` pattern